src/misc/debug.c

```diff
@@ -141,7 +141,54 @@ void debug_vprintf_trace(const char* func_name,
 			 va_list ap)
 
 {
+#ifdef USE_LOG_BACKEND
+	char fmt_copy[1024] = "";
+	strcpy(fmt_copy, fmt);
```

Comment:
could this not just be applied to tmp below?

Reply:
Yes, you're right, I've missed that for some reason.